src/storage/hummock_sdk/src/compaction_group/mod.rs

```diff
@@ -43,3 +43,105 @@ impl From<StaticCompactionGroupId> for CompactionGroupId {
         cg as CompactionGroupId
     }
 }
+
+pub mod group_split {
+    use std::cmp::Ordering;
+
+    use super::hummock_version_ext::{add_ssts_to_sub_level, insert_new_sub_level};
+    use crate::can_concat;
+    use crate::level::{Level, Levels};
+
+    pub fn merge_levels(left_levels: &mut Levels, right_levels: &mut Levels) {
+        let right_l0 = &right_levels.l0;
+
+        for right_sub_level in &right_l0.sub_levels {
+            let sub_level = right_sub_level.clone();
```

Comment:
IIUC, right_levels should be cleared eventually. Can we avoid `clone()` here but move?